spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -390,9 +395,39 @@ def _measure_axial_damage_ratio(self, im_data, p_lst, idx):
 
         # Save the maximum axial damage ratio
         self.measure_pd.loc[idx, 'max_axial_damage_ratio []'] = maximum_axial_damage_ratio
-        printv('  Maximum axial damage ratio : ' + str(np.round(maximum_axial_damage_ratio, 2)),
+        printv('  Maximum axial damage ratio: ' + str(np.round(maximum_axial_damage_ratio, 2)),
                self.verbose, type='info')
 
+    def _get_midsagittal_slice(self, idx):
```

Comment:
Completely agree! Thanks for the suggestion! It's great to get your dev feedback!
Function simplified in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4617/commits/a7b8654b98f91f92d2faf539a2d490f0bdfd1426

Reply:
> Also, as a side note, it might be good to rename self.sagittal_sc_slice to self.midsagittal_sc_slice_rpi since the RPI nature of the slice number is important for internal calculations.

I would maybe keep the current name because the function now (after https://github.com/spinalcordtoolbox/spinalcordtoolbox/commit/a7b8654b98f91f92d2faf539a2d490f0bdfd1426) get the midsagittal slice for both RPI and orig orientation. 
`self.midsagittal_sc_slice_rpi` sounds to me that the midsagittal slice would be obtained only for the RPI orientation.